transaction.go

```diff
@@ -782,49 +823,183 @@ func (tx *Transaction) _BuildTransaction(index int) (*services.Transaction, erro
 // Shared
 //
 
-// GetMaxTransactionFee returns the maximum transaction fee the operator (paying account) is willing to pay.
-func (tx *Transaction) GetMaxTransactionFee() Hbar {
-	return HbarFromTinybar(int64(tx.transactionFee))
+// GetSignedTransactionBodyBytes
+// TODO remove this method, it's not used in the sdk and it does not seem useful for the user
+func (tx *Transaction[T]) GetSignedTransactionBodyBytes(transactionIndex int) []byte {
```

Comment:
Would like to remove since it's unused and it's not present functionality in other sdks.